### PR TITLE
adjustments to the pack categories

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress

## Related Issues
fixes:[ link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5348?filter=-3)

## Description
make adjustments to the pack categories we offer in the Marketplace by removing:
- Messaging
- Authentication
- Deception

## Minimum version of Cortex XSOAR
- [x] 6.0.0

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Tests/Marketplace/approved_categories.json [hunk 1]
```diff
@@ -2,18 +2,14 @@
     "description": "Approved categories for use in pack_metadata.json and integration yml. To modify this file you will need a CODEOWNER to approve. Please make sure to trigger the sdk build in case you edit this list.",
 	"approved_list": [
 	"Analytics & SIEM",
-	"Authentication",
 	"Case Management",
 	"Cloud Services",
 	"Data Enrichment & Threat Intelligence",
 	"Database",
-	"Deception",
-	"Deception & Breach Simulation",
```
Comment (Shellyber) [inline Tests/Marketplace/approved_categories.json:11]: Why did you remove this also?

Comment (maimorag) [reply Tests/Marketplace/approved_categories.json:11]: I have no memory of removing this not sure how :|

**Review by Shellyber — APPROVED** [review]
